README.md

```diff
@@ -1,46 +1,113 @@
+  much technical knowledge.
+- **Demo Wallet:** The Demo wallet acts as the Issuer and Relying party frontend, showing the showcases, personas, scenarios and steps. This is what an end-user will see after
+  showcases have been created by the Showcase builder.
+- **Showcase API** The Showcase API is a REST API that serves as the technology neutral data storage between different components, of which the wallet and builder are the most
+  noticeable components. The API is credential technology neutral. It has no direct integrations with any credential platform/library.
+- **Traction Adapter** This is an adapter that translates the neutral API into AcaPY/Traction specific credential actions. It serves is the first of potentially many different
+  integrations points with future credential technologies.
+
+### 🏆 Key Benefits
+- **Easy of use:** The showcase builder is designed to be used by less technically inclined users, allowing them to experiment with different showcases
+- **Rapid prototyping:** The showcase builder allows you to rapidly create an entire new showcase, or add new scenarios to existing showcases. This allows for rapid experimentation and customization
+- **Multi-tenant:** The showcase builder, API and wallet are designed as multi-tenant solutions, allowing to make the tool available to different departments, or even external customers
+- **Flexibility:** Adapters can be swapped or extended to support different credential formats (e.g., SD-JWT, OID4VCI) and wallets in the future.
+- **Resilience:** Durable messaging to handle temporary outages without data loss.
+
+## 📁 Project Structure
+
+```
+apps/
+├── bc-wallet-api-server/        # This is the Showcase REST API providing the credential agnostic data model
+├── bc-wallet-demo-server/       # The server/backend of the web wallet delegating to AcaPy and Traction
+├── bc-wallet-demo-web/          # The frontend of the web wallet showing the actual showcases, personas and scenarios
+├── bc-wallet-showcase-creator/  # The frontend to design and manage showcases, scenarios, credential definitions and personas
+charts/                          # Helm charts for deployment to Kubernetes
+packages/
+├── bc-wallet-openapi/           # The Showcase OpenAPI definition for the REST API and models
+├── bc-wallet-traction-adapter   # The AcaPY/Traction integration (asynchronous using a message broker)
+resources/                       # Documentation resources
+├── DEVELOPER                    # Developer docs
+├── DEVOPS/                      # DevOps docs
+└── README.md                    # Project documentation (this file)
+```
+## 📦 Package Management
+
+This project uses **pnpm** and **turbo** for monorepo and package management.
+
+### Make sure pnpm is installed
+
+````shell
+npm -g install pnpm
+````
+
+### Install Dependencies
+
```

Comment:
trivial, but until we got the `bc-wallet-openapi` as an external package, before the `pnpm install`
we might run 
`pnpm generate:models`
and make sure we have maven available https://maven.apache.org/download.cgi